### PR TITLE
document electron implementation option

### PR DESCRIPTION
**Pull request by lstreckeisen**

### CODE REVIEW

File: SA/design_concept/content/poc/electron.typ [hunk 1]
```diff
@@ -0,0 +1,14 @@
+= Electron JS
+Electron JS is a framework for creating desktop apps using HTML, JS and CSS.
+This is achieved through the combination of chromium and Node.js.
+Electron allows the creation of desktop apps in web-app style, which means that, as long as no native OS resources are needed, the app is automatically capable of running on different OS platforms.
+@electron-js
+
+Pros:
```
Comment (lstreckeisen) [inline SA/design_concept/content/poc/electron.typ:7]: @fl3pp Do you have anything to add here?

Comment (lstreckeisen) [reply SA/design_concept/content/poc/electron.typ:7]: I'll leave it as it is for now and maybe add an example later.

File: SA/design_concept/content/poc/implementation_options.typ [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#import "../../../style.typ": include_section
+
+= Implementation Options
```
[the remark this answers is not in the document]
Comment (lstreckeisen) [reply SA/design_concept/content/poc/implementation_options.typ:3]: This was intended to be preparation for that "later section". I'm not sure about the naming either, but we can adjust and restructure anytime.

[the remark this answers is not in the document]
Comment (fl3pp) [reply SA/design_concept/content/poc/implementation_options.typ:3]: yes, that makes sense 👍

**Review by fl3pp — APPROVED** [review]

**Pull request merged**